pkg/watermark/publish/publisher.go

```diff
@@ -242,7 +248,13 @@ func (p *publish) loadLatestFromStore() wmb.Watermark {
 
 // GetLatestWatermark returns the latest watermark for that processor.
 func (p *publish) GetLatestWatermark() wmb.Watermark {
-	return p.GetHeadWM()
+	var latestWatermark = wmb.Watermark(time.UnixMilli(-1))
```

Comment:
Same here, we may want to create a const for `-1` and use it for any watermark initialization.